core/src/main/java/io/questdb/griffin/InsertStatementImpl.java

```diff
@@ -107,7 +107,7 @@ public long execute() {
                 InsertRowImpl row = insertRows.get(i);
                 row.append(writer);
             }
-            return 1;
+            return insertRows.size();
```

Comment:
This used to return 1 not matter what, it may now return 0, just noting.

Reply:
Yes, this was a bug I fixed . Multi-row insert statement was always reporting 1 as number of inserted rows .